schema.md

```diff
@@ -0,0 +1,43 @@
+# Cómo utilizar `schema.org`
+
+Schema.org provee una colección de vocabularios compartidos 
+que puedes utilizar para que las máquinas de búsqueda puedan entender tu página web.
+
+Vamos a utilizar un ejemplo:
+
+![schema](images/schema.png)
+
+`itemscope` indica que, lo que está dentro del bloque `<div>...</div>` 
+es un elemento en particular.
+
+`itemtype` especifica el tipo de elemento que se está utilizando.
+se coloca después del `itemscope`.
+
+`itemprop` especifica las propiedades del elemento. 
+
+===============================================================================
```

Comment:
Markdown has a way to generate horizontal rules, `----`. 

----

But I think that instead of using horizontal rules, we can use headers (h2 for the main sections) to break apart the work and it will help readability of the document. 

What I read the document, I sense 3 sections 

## What is schema.org?

## Tips for implementing/using schema.org

## Testing schema.org

What do you think?